frontend/src/metabase/schema.js

```diff
@@ -70,6 +71,10 @@ export const SnippetCollectionSchema = new schema.Entity("snippetCollections");
 export const TimelineSchema = new schema.Entity("timelines");
 export const TimelineEventSchema = new schema.Entity("timelineEvents");
 
+DashboardSchema.define({
+  dashcards: [DashcardSchema],
+});
+
```

Comment:
Why do we need to normalize dashboard cards now? Not sure I've noticed a place where it's used

Reply:
I didn't think this was necessary either but I tried it and it fixed a bunch of e2e tests. But after hearing about your bug I suspect it fixed them because of a race condition. I'll remove this